jsettlers.graphics.swing/resources/defaultConfig.prp

```diff
@@ -1,3 +1,5 @@
-# Default settings.
-resources-folder = ../jsettlers.common/resources/
-settlers-folder = .
\ No newline at end of file
+#Default Settings
+#Sat May 30 17:47:34 BST 2015
+resources-folder=../jsettlers.common/resources/
+settlers-folder=.
+commit_info=86d82dd
```

Comment:
config.prp is intended to be modifiable by the user. This is no good place to store the commit hash.